lte/protos/session_manager.proto

```diff
@@ -933,17 +933,6 @@ message RatSpecificContextAccess {
 // e.g. to determine whether a resource is available for downlink media of a media stream of an SDP media
 // description provided by the UE in an IMS session.
 
-message AmbrUnit {
-  enum AmbrUnitType {
-   Kbps_0 = 0;
-   Kbps_1 = 1;
-   Kbps_4 = 2;
-   Kbps_16 = 3;
-   Kbps_64 = 4;
-  }
-  AmbrUnitType ambr_unit_type = 1;
-}
-
```

Comment:
removing this looks good. 

For future cases, remember we can pass around the raw value. Let the service that uses that value to decode it without the need to go through 3gpp structures